tests/e2e/negative_day_one_test.go

```diff
@@ -841,4 +841,24 @@ var _ = Describe("Negative Tests", Ordered, func() {
 			restoreSubnetTagging()
 		})
 	})
+
+	var _ = Describe("The EOL OCP version validation", ci.NonHCPCluster, ci.Day1Negative, func() {
```

Comment:
Can you test it against HCP as well ? It should work IMO

Reply:
@radtriste the test can work with HCP cluster, remove d ci.NonHCPCluster label.